windows/src/developer/TIKE/dialogs/languages/Keyman.Developer.UI.UfrmSelectBCP47Language.pas

```diff
@@ -125,9 +125,9 @@ procedure TfrmSelectBCP47Language.RefreshPreview;
   msg: string;
 begin
   editBCP47Code.Text := tag.Tag;
-  lblLanguageName.Caption := tag.Language; // TODO: BCP47: Lookup language name
-  lblScriptName.Caption := tag.Script; // TODO: BCP47: Lookup script name
-  lblRegionName.Caption := tag.Region; // TODO: BCP47: Lookup region name
+  lblLanguageName.Caption := tag.Language; // TODO: BCP47: <DeveloperBCP47LanguageLookup> Lookup language name
+  lblScriptName.Caption := tag.Script; // TODO: BCP47: <DeveloperBCP47LanguageLookup> Lookup script name
+  lblRegionName.Caption := tag.Region; // TODO: BCP47: <DeveloperBCP47LanguageLookup> Lookup region name
   cmdOK.Enabled := tag.IsValid(msg);
```

Comment:
Will you need to handle `Variant` or `PrivateUse`? I'm thinking for "fonipa"

Reply:
Yes, this is an issue because none of the target operating systems support it. However, this work can be part of the language picker PR that is yet to come.